includes/class-admin-settings.php

```diff
@@ -130,11 +131,37 @@ public function reset_admin_notice() {
 			isset( $_GET['reset-success-nonce'] ) &&
 			wp_verify_nonce( sanitize_key( $_GET['reset-success-nonce'] ), 'aspireupdate-reset-success-nonce' )
 		) {
-			echo '<div class="notice notice-success is-dismissible"><p>' . esc_html__( 'Settings have been reset to default.', 'AspireUpdate' ) . '.</p></div>';
+			add_settings_error(
+				'aspireupdate_settings_reset',
+				'aspireupdate_settings_reset',
+				esc_html__( 'Settings have been reset to default.', 'AspireUpdate' ),
+				'success'
+			);
+			settings_errors( 'aspireupdate_settings_reset' );
 			delete_site_option( 'aspireupdate-reset' );
 		}
 	}
 
+	/**
+	 * The Admin Notice to convey settings have been successsfully saved.
+	 *
+	 * @return void
+	 */
+	public function settings_saved_admin_notice() {
```

Comment:
I merged both functions but didn't go about creating separate functions for the conditions, it will just add up new functions within the class with very little scope of reuse.